new file Creative Commons Licences

broughd-qld opened the pull request:
Uploading the Creative Commons licences for use in the DES catalogue.
Started with a licence file from http://registry.it.csiro.au/licence and reduced to just using the CC licences.
File has passed testing in SKOS testing tool and SKOS Explorer.
@KellyVance Is is possible to have this one done by 20 November 2020?

KellyVance commented on vocabularies/licence.ttl:
```diff
@@ -0,0 +1,52 @@
+@prefix rdfs:  <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos:  <http://www.w3.org/2004/02/skos/core#> .
+@prefix reg:   <http://purl.org/linked-data/registry#> .
+@prefix dct:   <http://purl.org/dc/terms/> .
+@prefix ldp:   <http://www.w3.org/ns/ldp#> .
+@prefix owl:   <http://www.w3.org/2002/07/owl#> .
+@prefix rr-org: <http://registry.it.csiro.au/structure/org/> .
+
```
```suggestion
@prefix rdfs:  <http://www.w3.org/2000/01/rdf-schema#> .
@prefix skos:  <http://www.w3.org/2004/02/skos/core#> .
@prefix reg:   <http://purl.org/linked-data/registry#> .
@prefix dct:   <http://purl.org/dc/terms/> .
@prefix ldp:   <http://www.w3.org/ns/ldp#> .
@prefix owl:   <http://www.w3.org/2002/07/owl#> .
@prefix rr-org: <http://registry.it.csiro.au/structure/org/> .
@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
```

Will need xsd prefix declaration for modified date.

KellyVance commented on vocabularies/licence.ttl:
```diff
@@ -0,0 +1,52 @@
+        reg:delegationTarget  <https://creativecommons.org/licenses/by/4.0/> ;
+        reg:forwardingCode    "307" .
+
+<http://registry.it.csiro.au/licence>
+        a                      ldp:Container , reg:Registerd , skos:Concept ;
+        skos:inScheme <http://registry.it.csiro.au/licence> ;
+        skos:topConceptOf <http://registry.it.csiro.au/licence> ;
+        skos:prefLabel        "Licences"@en ;
+        rdfs:label             "Licences"@en ;
+        rdfs:member            <http://registry.it.csiro.au/licence/cc-by-4.0> , <http://registry.it.csiro.au/licence/cc-by-nd-4.0> , <http://registry.it.csiro.au/licence/cc-by-sa-4.0> ;
+        skos:hasTopConcept     <http://registry.it.csiro.au/licence/cc-by-4.0> , <http://registry.it.csiro.au/licence/cc-by-nd-4.0> , <http://registry.it.csiro.au/licence/cc-by-sa-4.0> ;
+        dct:description        "Licences for use of software, data, documents and other resources"@en ;
+        dct:modified           "2015-08-10T10:18:29.275Z"^^xsd:dateTime ;
+        reg:owner              rr-org:csiro ;
+        owl:versionInfo        1 ;
+        ldp:hasMemberRelation  rdfs:member .
```
```suggestion
<http://registry.it.csiro.au/licence>
        a                      ldp:Container , reg:Registerd , skos:ConceptScheme ;
        skos:prefLabel        "Creative Commons Licences"@en ;
        rdfs:label            "Creative Commons Licences"@en ;
        rdfs:member           <http://registry.it.csiro.au/licence/cc-by-4.0> , 
                                        <http://registry.it.csiro.au/licence/cc-by-nd-4.0> ,
                                        <http://registry.it.csiro.au/licence/cc-by-sa-4.0> ;
        skos:hasTopConcept    <http://registry.it.csiro.au/licence/cc-by-4.0> ,
                                         <http://registry.it.csiro.au/licence/cc-by-nd-4.0> ,
                                         <http://registry.it.csiro.au/licence/cc-by-sa-4.0> ;
        dct:description        "Licences for use of software, data, documents and other resources"@en ;
        skos:definition        "Licences for use of software, data, documents and other resources"@en ;
        dct:modified           "2015-08-10T10:18:29.275Z"^^xsd:dateTime ;
        reg:owner              rr-org:csiro ;
        owl:versionInfo        1 ;
        ldp:hasMemberRelation  rdfs:member .
```

This looks like the Concept Scheme rather than a concept.
Therefore shouldn't be a topConcept of itself

KellyVance commented on vocabularies/licence.ttl:
```diff
@@ -0,0 +1,52 @@
+        skos:topConceptOf <http://registry.it.csiro.au/licence> ;
+        rdfs:label            "Creative Commons Attribution-NoDerivatives 4.0 International"@en ;
+        skos:prefLabel        "Creative Commons Attribution-NoDerivatives 4.0 International"@en ;
+        dct:description       "You are free to: Share - copy and redistribute the material in any medium or format for any purpose, even commercially. The licensor cannot revoke these freedoms as long as you follow the licence terms. Under the following terms: Attribution - You must give appropriate credit, provide a link to the licence, and indicate if changes were made. You may do so in any rea ...
```
```suggestion
        dct:description       "You are free to: Share - copy and redistribute the material in any medium or format for any purpose, even commercially. The licensor cannot revoke these freedoms as long as you follow the licence terms. Under the following terms: Attribution - You must give appropriate credit, provide a link to the licence, and indicate if changes were made. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use. NoDerivatives - If you remix, transform, or build upon the material, you may not distribute the modified material. No additional restrictions - You may not apply legal terms or technological measures that legally restrict others from doing anything the licence permits Notices - You do not have to comply with the licence for elements of the material in the public domain or where your use is permitted by an applicable exception or limitation. No warranties are given. The licence may not give you all of the permissions necessary for your intended use. For example, other rights such as publicity, privacy, or moral rights may limit how you use the material."@en ;
        skos:definition       "You are free to: Share - copy and redistribute the material in any medium or format for any purpose, even commercially. The licensor cannot revoke these freedoms as long as you follow the licence terms. Under the following terms: Attribution - You must give appropriate credit, provide a link to the licence, and indicate if changes were made. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use. NoDerivatives - If you remix, transform, or build upon the material, you may not distribute the modified material. No additional restrictions - You may not apply legal terms or technological measures that legally restrict others from doing anything the licence permits Notices - You do not have to comply with the licence for elements of the material in the public domain or where your use is permitted by an applicable exception or limitation. No warranties are given. The licence may not give you all of the permissions necessary for your intended use. For example, other rights such as publicity, privacy, or moral rights may limit how you use the material."@en ;
```

I have to admit I'm not sure if our VocPrez instance will pick up on dct:description, or if it will only display a skos:definition. You may need both for each concept if that is the case. Do you know the answer, @DavidCrosswellGSQ?

broughd-qld replied:
@KellyVance could we stick with calling it 'Licence' or something similar. We will likely have additional licence types to add in the future that won't necessarily be Creative Commons.

KellyVance replied:
That might be a good approach. Can always create a _Creative Commons_ collection to pre-empt any potential expansion.

broughd-qld replied:
I can't see how to commit part of the suggestion, will commit entirely and then revert the label back to _licence_

johnmckellar replied:
Sorry, I can't comment on any of this. But, I assume that any licence/s that is/are set-up would need to be run past the departmental legal people. Back in 2013, when I published a paper in a Taylor & Francis journal who would normally hold the copyright, I overrode this to obtain Crown copyright, which was not unusual. But, I was blown away by the detail that the legal people drew up.

broughd-qld replied:
We use CC licences for just about everything. If we do have other specialised licences they are done by the business groups and with legal input.

johnmckellar replied:
All good. I was just noting Vance's comment that we will likely have additional licence types in the future that won't necessarily be Creative Commons.

KellyVance reviewed (changes requested):
Two changes to make, and one question that DC should be able to answer.

KellyVance reviewed (approved):
Looks good to me.

johnmckellar commented:
Looks seemingly OK

DavidCrosswellGSQ commented on vocabularies/licence.ttl:
```diff
@@ -0,0 +1,56 @@
+@prefix rdfs:  <http://www.w3.org/2000/01/rdf-schema#> .
```
```suggestion
@prefix copyl: <http://linked.data.gov.au/def/copyright-licence/>
@prefix rdfs:  <http://www.w3.org/2000/01/rdf-schema#> .
```

broughd-qld replied:
agree

DavidCrosswellGSQ replied:
licence document is good.

DavidCrosswellGSQ commented on vocabularies/licence.ttl:
```diff
@@ -0,0 +1,56 @@
+<http://registry.it.csiro.au/licence>
+        a                      ldp:Container , reg:Registerd , skos:ConceptScheme ;
+        skos:prefLabel        "Licences"@en ;
+        rdfs:label            "Licences"@en ;
```
```suggestion
        rdfs:label            "Copyright Licences"@en ;
```

DavidCrosswellGSQ commented on vocabularies/licence.ttl:
```diff
@@ -0,0 +1,56 @@
+
+<http://registry.it.csiro.au/licence>
+        a                      ldp:Container , reg:Registerd , skos:ConceptScheme ;
+        skos:prefLabel        "Licences"@en ;
```
```suggestion
```

DavidCrosswellGSQ commented on vocabularies/licence.ttl:
```diff
@@ -0,0 +1,56 @@
+        reg:delegationTarget  <https://creativecommons.org/licenses/by/4.0/> ;
+        reg:forwardingCode    "307" .
+
+<http://registry.it.csiro.au/licence>
```
```suggestion
<http://linked.data.gov.au/def/copyright-licence/>
```

DavidCrosswellGSQ reviewed (changes requested):
Let's make this a linked.data.gov.au vocabulary.
This vocab should be a copyright licence vocabulary that contains the types used in Australian works (as a start(.
If DES only wants to display the CC licences in their dropdown, then this can be done in the SPARQL query.

broughd-qld commented:
> Let's make this a linked.data.gov.au vocabulary.
> This vocab should be a copyright licence vocabulary that contains the types used in Australian works (as a start(.
> If DES only wants to display the CC licences in their dropdown, then this can be done in the SPARQL query.

@DavidCrosswellGSQ I'll make the changes to switch to linked.data.gov.au. I'm not going to accept the copyright item, as they aren't really copyright and are meant to be about permitted and licenced use of resources. The list can deal with digital and non-digital. Dublin Core uses the term **_Licence Document_** would you be comfortable with that?

broughd-qld commented:
@DavidCrosswellGSQ Have moved to linked.data.gov.au and changed to licence-document vocabulary name (which matches Dublin Core for the range of a _licence_)

Did a bit of clean up to creator and added editoralnote to concept scheme.

DavidCrosswellGSQ commented on vocabularies/licence.ttl:
```diff
@@ -0,0 +1,57 @@
+        reg:delegationTarget  <https://creativecommons.org/licenses/by/4.0/> ;
+        reg:forwardingCode    "307" .
+
+<http://linked.data.gov.au/def/licence-document>
```
Can you please put the ConceptScheme at the top of this document (line 10).
This is a consistent pattern that we have implemented.
Additionally (for longer vocabs) I have noticed that VocPrez runs into issues loading vocabs if the concept scheme is lower.

broughd-qld replied:
done

DavidCrosswellGSQ commented on vocabularies/licence.ttl:
```diff
@@ -0,0 +1,57 @@
+
+<http://linked.data.gov.au/def/licence-document>
+        a                      ldp:Container , reg:Registerd , skos:ConceptScheme ;
+        skos:prefLabel        "Licences"@en ;
```
```suggestion
        skos:prefLabel        "Licence Document"@en ;
```

DavidCrosswellGSQ commented on vocabularies/licence.ttl:
```diff
@@ -0,0 +1,57 @@
+<http://linked.data.gov.au/def/licence-document>
+        a                      ldp:Container , reg:Registerd , skos:ConceptScheme ;
+        skos:prefLabel        "Licences"@en ;
+        rdfs:label            "Licences"@en ;
```
Is there a reason that you are using rdfs:label in the vocab?

broughd-qld replied:
Only because it was in the CSIRO source - will delete.

DavidCrosswellGSQ reviewed (changes requested):
@broughd-qld - some requested changes.
Thanks, DC

broughd-qld commented:
@DavidCrosswellGSQ - changes made as per request.
